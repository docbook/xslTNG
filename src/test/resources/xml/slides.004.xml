<book xmlns="http://docbook.org/ns/docbook"
      xmlns:xlink="http://www.w3.org/1999/xlink"
      xmlns:xi="http://www.w3.org/2001/XInclude"
      version="5.1">
<info>
<title>Unit test: slides.004</title>
<author><personname>Norm Tovey-Walsh</personname></author>
<meta xmlns="http://www.w3.org/1999/xhtml"
      name="localStorage.key" content="windowLocation"/>
<meta xmlns="http://www.w3.org/1999/xhtml"
      name="progressiveReveal.key" content="progressiveReveal"/>
</info>

<article role="foil">
<title>The Speaker</title>
<itemizedlist>
<listitem><para>Did a thing
</para></listitem>
<listitem><para>Has a thing
</para></listitem>
<listitem><para>Also did another thing
</para></listitem>
</itemizedlist>
</article>

<article role="foil">
<title>You?</title>
<para role="speaker-notes">These are speaker notes</para>
<itemizedlist role="speaker-notes">
<listitem><para>Norm
</para></listitem>
<listitem><para>Stuff
</para></listitem>
<listitem><para>Things
</para></listitem>
</itemizedlist>

<itemizedlist>
<listitem><para>What is your name?
</para></listitem>
<listitem><para>What do you do?
</para></listitem>
<listitem><para>What are you hoping to get out of this course?
</para></listitem>
</itemizedlist>
</article>

<article role="foil">
<title>What is this?</title>

<para>Sometimes it’s useful to present training materials in “slide”
form. That is one page at a time, in the browser, projected in a
classroom or presented over the web.</para>

<para>That’s a lot like paging through a book in the “chunked” view, so we’re
going to abuse the heck out of that markup.</para>
</article>

<article role="foil">
<title>A typical slide</title>
<itemizedlist>
<listitem><para>Slides often wind up being just a list of bullet points
</para></listitem>
<listitem><para>This probably not the ideal presentation technique
</para></listitem>
<listitem><para>But it’s typical never-the-less.
</para></listitem>
<listitem><para>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</para></listitem>
</itemizedlist>
</article>

<article role="foil">
<title>Sequential presentation</title>
<itemizedlist role="reveal">
<listitem><para>Slides often wind up being just a list of bullet points
</para></listitem>
<listitem><para>This probably not the ideal presentation technique
</para></listitem>
<listitem><para>But it’s typical never-the-less.
</para></listitem>
<listitem><para>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</para></listitem>
</itemizedlist>
</article>

<article role="foil">
<title>Sequential presentation 2</title>
<itemizedlist>
<listitem><para>Slides often wind up being just a list of bullet points
</para></listitem>
<listitem><para>This probably not the ideal presentation technique
</para></listitem>
<listitem><para>But it’s typical never-the-less.
</para></listitem>
<listitem><para>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</para></listitem>
</itemizedlist>
<itemizedlist role="reveal">
<listitem><para>Slides often wind up being just a list of bullet points
</para></listitem>
<listitem><para>This probably not the ideal presentation technique
</para></listitem>
<listitem><para>But it’s typical never-the-less.
</para></listitem>
<listitem><para>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</para></listitem>
</itemizedlist>
<itemizedlist>
<listitem><para>Slides often wind up being just a list of bullet points
</para></listitem>
<listitem><para>This probably not the ideal presentation technique
</para></listitem>
<listitem><para>But it’s typical never-the-less.
</para></listitem>
<listitem><para>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</para></listitem>
</itemizedlist>
</article>

<article>
<title>Program listing</title>
<programlisting language="xml"><![CDATA[<article role="foil">
<title>Sequential presentation 2</title>
<itemizedlist>
<listitem><para>Slides often wind up being just a list of bullet points
</para></listitem>
<listitem><para>This probably not the ideal presentation technique
</para></listitem>
<listitem><para>But it’s typical never-the-less.
</para></listitem>
<listitem><para>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</para></listitem>
</itemizedlist>
<itemizedlist role="reveal">
<listitem><para>Slides often wind up being just a list of bullet points
</para></listitem>
<listitem><para>This probably not the ideal presentation technique
</para></listitem>
<listitem><para>But it’s typical never-the-less.
</para></listitem>
<listitem><para>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</para></listitem>
</itemizedlist>
<itemizedlist>
<listitem><para>Slides often wind up being just a list of bullet points
</para></listitem>
<listitem><para>This probably not the ideal presentation technique
</para></listitem>
<listitem><para>But it’s typical never-the-less.
</para></listitem>
<listitem><para>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</para></listitem>
</itemizedlist>
</article>]]></programlisting>
</article>

<article>
<title>Thank you</title>
<para>Thank you!</para>
</article>

</book>
