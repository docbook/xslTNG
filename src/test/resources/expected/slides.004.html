<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><title>Unit test: slides.004</title><link href="./css/pygments.css" rel="stylesheet"/><meta name="viewport" content="width=device-width, initial-scale=1.0"/><link href="https://purl.org/dc/elements/1.1/" rel="schema.dc"/><meta content="2011-04-22T17:02:00-06:00" name="dc.modified"/><meta content="DocBook xslTNG" name="generator"/><link href="./css/docbook.css" rel="stylesheet" media="screen"/><meta name="localStorage.key" content="windowLocation"/><meta name="progressiveReveal.key" content="progressiveReveal"/><script src="./js/copy-verbatim.js" defer="defer"></script></head><body class="home"><nav class="top"></nav><main><article class="book division"><header><h1>Unit test: slides.004</h1><div class="author"><h3><span class="first-last personname">Norm Tovey-Walsh</span></h3></div></header><div class="list-of-titles"><div class="lot toc"><div class="title">Table of Contents</div><ul class="toc"><li><a href="#R_article1">The Speaker</a></li><li><a href="#R_article2">You?</a></li><li><a href="#R_article3">What is this?</a></li><li><a href="#R_article4">A typical slide</a></li><li><a href="#R_article5">Sequential presentation</a></li><li><a href="#R_article6">Sequential presentation 2</a></li><li><a href="#R_article7">Program listing</a></li><li><a href="#R_article8">Thank you</a></li></ul></div></div><section id="R_article1" class="article component foil"><header><h1>The Speaker</h1></header><ul class="itemizedlist"><li><p>Did a thing
</p></li><li><p>Has a thing
</p></li><li><p>Also did another thing
</p></li></ul></section><section id="R_article2" class="article component foil"><header><h1>You?</h1></header><p class="speaker-notes">These are speaker notes</p><ul class="itemizedlist speaker-notes"><li><p>Norm
</p></li><li><p>Stuff
</p></li><li><p>Things
</p></li></ul><ul class="itemizedlist"><li><p>What is your name?
</p></li><li><p>What do you do?
</p></li><li><p>What are you hoping to get out of this course?
</p></li></ul></section><section id="R_article3" class="article component foil"><header><h1>What is this?</h1></header><p>Sometimes it’s useful to present training materials in “slide”
form. That is one page at a time, in the browser, projected in a
classroom or presented over the web.</p><p>That’s a lot like paging through a book in the “chunked” view, so we’re
going to abuse the heck out of that markup.</p></section><section id="R_article4" class="article component foil"><header><h1>A typical slide</h1></header><ul class="itemizedlist"><li><p>Slides often wind up being just a list of bullet points
</p></li><li><p>This probably not the ideal presentation technique
</p></li><li><p>But it’s typical never-the-less.
</p></li><li><p>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</p></li></ul></section><section id="R_article5" class="article component foil"><header><h1>Sequential presentation</h1></header><ul class="itemizedlist reveal"><li><p>Slides often wind up being just a list of bullet points
</p></li><li><p>This probably not the ideal presentation technique
</p></li><li><p>But it’s typical never-the-less.
</p></li><li><p>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</p></li></ul></section><section id="R_article6" class="article component foil"><header><h1>Sequential presentation 2</h1></header><ul class="itemizedlist"><li><p>Slides often wind up being just a list of bullet points
</p></li><li><p>This probably not the ideal presentation technique
</p></li><li><p>But it’s typical never-the-less.
</p></li><li><p>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</p></li></ul><ul class="itemizedlist reveal"><li><p>Slides often wind up being just a list of bullet points
</p></li><li><p>This probably not the ideal presentation technique
</p></li><li><p>But it’s typical never-the-less.
</p></li><li><p>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</p></li></ul><ul class="itemizedlist"><li><p>Slides often wind up being just a list of bullet points
</p></li><li><p>This probably not the ideal presentation technique
</p></li><li><p>But it’s typical never-the-less.
</p></li><li><p>(This is DocBook, you can put anything you want on the slides,
even pretty cat pictures if you want.)
</p></li></ul></section><section id="R_article7" class="article component"><header><h1>Program listing</h1></header><div class="pre-wrap highlight" db-startinglinenumber="1" db-numberoflines="36"><pre class="language-xml long numbered programlisting verbatim verblines"><span class="line" db-line="1"><span class="ln"> 1 <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;article</span><span class="w"> </span><span class="na">role=</span><span class="s">"foil"</span><span class="nt">&gt;</span></code></span></span>
<span class="line" db-line="2"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;title&gt;</span>Sequential<span class="w"> </span>presentation<span class="w"> </span>2<span class="nt">&lt;/title&gt;</span></code></span></span>
<span class="line" db-line="3"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;itemizedlist&gt;</span></code></span></span>
<span class="line" db-line="4"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>Slides<span class="w"> </span>often<span class="w"> </span>wind<span class="w"> </span>up<span class="w"> </span>being<span class="w"> </span>just<span class="w"> </span>a<span class="w"> </span>list<span class="w"> </span>of<span class="w"> </span>bullet<span class="w"> </span>points</code></span></span>
<span class="line" db-line="5"><span class="ln"> 5 <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="6"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>This<span class="w"> </span>probably<span class="w"> </span>not<span class="w"> </span>the<span class="w"> </span>ideal<span class="w"> </span>presentation<span class="w"> </span>technique</code></span></span>
<span class="line" db-line="7"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="8"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>But<span class="w"> </span>it’s<span class="w"> </span>typical<span class="w"> </span>never-the-less.</code></span></span>
<span class="line" db-line="9"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="10"><span class="ln">10 <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>(This<span class="w"> </span>is<span class="w"> </span>DocBook,<span class="w"> </span>you<span class="w"> </span>can<span class="w"> </span>put<span class="w"> </span>anything<span class="w"> </span>you<span class="w"> </span>want<span class="w"> </span>on<span class="w"> </span>the<span class="w"> </span>slides,</code></span></span>
<span class="line" db-line="11"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code>even<span class="w"> </span>pretty<span class="w"> </span>cat<span class="w"> </span>pictures<span class="w"> </span>if<span class="w"> </span>you<span class="w"> </span>want.)</code></span></span>
<span class="line" db-line="12"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="13"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/itemizedlist&gt;</span></code></span></span>
<span class="line" db-line="14"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;itemizedlist</span><span class="w"> </span><span class="na">role=</span><span class="s">"reveal"</span><span class="nt">&gt;</span></code></span></span>
<span class="line" db-line="15"><span class="ln">15 <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>Slides<span class="w"> </span>often<span class="w"> </span>wind<span class="w"> </span>up<span class="w"> </span>being<span class="w"> </span>just<span class="w"> </span>a<span class="w"> </span>list<span class="w"> </span>of<span class="w"> </span>bullet<span class="w"> </span>points</code></span></span>
<span class="line" db-line="16"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="17"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>This<span class="w"> </span>probably<span class="w"> </span>not<span class="w"> </span>the<span class="w"> </span>ideal<span class="w"> </span>presentation<span class="w"> </span>technique</code></span></span>
<span class="line" db-line="18"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="19"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>But<span class="w"> </span>it’s<span class="w"> </span>typical<span class="w"> </span>never-the-less.</code></span></span>
<span class="line" db-line="20"><span class="ln">20 <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="21"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>(This<span class="w"> </span>is<span class="w"> </span>DocBook,<span class="w"> </span>you<span class="w"> </span>can<span class="w"> </span>put<span class="w"> </span>anything<span class="w"> </span>you<span class="w"> </span>want<span class="w"> </span>on<span class="w"> </span>the<span class="w"> </span>slides,</code></span></span>
<span class="line" db-line="22"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code>even<span class="w"> </span>pretty<span class="w"> </span>cat<span class="w"> </span>pictures<span class="w"> </span>if<span class="w"> </span>you<span class="w"> </span>want.)</code></span></span>
<span class="line" db-line="23"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="24"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/itemizedlist&gt;</span></code></span></span>
<span class="line" db-line="25"><span class="ln">25 <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;itemizedlist&gt;</span></code></span></span>
<span class="line" db-line="26"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>Slides<span class="w"> </span>often<span class="w"> </span>wind<span class="w"> </span>up<span class="w"> </span>being<span class="w"> </span>just<span class="w"> </span>a<span class="w"> </span>list<span class="w"> </span>of<span class="w"> </span>bullet<span class="w"> </span>points</code></span></span>
<span class="line" db-line="27"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="28"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>This<span class="w"> </span>probably<span class="w"> </span>not<span class="w"> </span>the<span class="w"> </span>ideal<span class="w"> </span>presentation<span class="w"> </span>technique</code></span></span>
<span class="line" db-line="29"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="30"><span class="ln">30 <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>But<span class="w"> </span>it’s<span class="w"> </span>typical<span class="w"> </span>never-the-less.</code></span></span>
<span class="line" db-line="31"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="32"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;listitem&gt;&lt;para&gt;</span>(This<span class="w"> </span>is<span class="w"> </span>DocBook,<span class="w"> </span>you<span class="w"> </span>can<span class="w"> </span>put<span class="w"> </span>anything<span class="w"> </span>you<span class="w"> </span>want<span class="w"> </span>on<span class="w"> </span>the<span class="w"> </span>slides,</code></span></span>
<span class="line" db-line="33"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code>even<span class="w"> </span>pretty<span class="w"> </span>cat<span class="w"> </span>pictures<span class="w"> </span>if<span class="w"> </span>you<span class="w"> </span>want.)</code></span></span>
<span class="line" db-line="34"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/para&gt;&lt;/listitem&gt;</span></code></span></span>
<span class="line" db-line="35"><span class="ln">35 <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/itemizedlist&gt;</span></code></span></span>
<span class="line" db-line="36"><span class="ln">   <span class="nsep">|</span></span><span class="ld"><code><span class="nt">&lt;/article&gt;</span></code></span></span>
</pre></div></section><section id="R_article8" class="article component"><header><h1>Thank you</h1></header><p>Thank you!</p></section></article></main><nav class="bottom"></nav></body></html>